config/default.yml

```diff
@@ -199,6 +199,16 @@ Rails/ContentTag:
   Exclude:
     - app/models/**/*.rb
 
+Rails/ControllerTesting:
+  Description: 'Use `ActionDispatch::IntegrationTest` instead of `ActionController::TestCase`.'
+  Reference:
+    - 'https://api.rubyonrails.org/classes/ActionController/TestCase.html'
+  Enabled: true
+  VersionAdded: '<<next>>'
+  Include:
+    - test/**/*.rb
```

Comment:
We don't always follow this in cops, but using:
```
    - '**/test/**/*'
```
would allow inspection for nested in-project engines, gems etc.